api/api_full.go

```diff
@@ -918,6 +919,20 @@ type MinerPower struct {
 	HasMinPower bool
 }
 
+type MinerInfo struct {
```

Comment:
move this with the other typedefs?

Reply:
The JSON-RPC API struct is different to state struct, I think. I think it's fine here.

Reply:
I meant to `api/types.go`, that's the logical place where it belongs.

Reply:
A bunch of the api types are already in api_full.go, but I think i agree with @vyzo -- moving.